lib/vdsm/virt/vm.py

```diff
@@ -6106,6 +6106,16 @@ def onDeviceRemoved(self, device_alias):
 
     # Accessing storage
 
+    def prune_bitmaps(self, domainID, imageID, topID, baseID):
+        """
+        Prune stale bitmaps from the base volume.
+        """
+        self.cif.irs.prune_bitmaps(
```

Comment:
You are not checking errors - this will not raise on errors!

Check how error are handled in measure() bellow.

Reply:
It will, if it fails it will raise an `exception.RemoveBitmapError` from the bitmaps module [1].

[1] https://github.com/oVirt/vdsm/blob/master/lib/vdsm/storage/bitmaps.py#L122

Reply:
The exception is not raised in the caller. HSM is wrapped by Dispatcher, which warps public methods to return status dict instead of raising.

Reply:
Ah! I see it now, I though `res` in measure was the return of the hsm measure (dict from the parsed json), but now I see the dispatcher in the decorator. Got it.